book_my_ticket/sources/book_my_ticket.move

```diff
@@ -251,11 +357,70 @@ module TicketProject::BookMyTicket {
         verify
     }
 
-#[test]
 
-public fun test_check(){
-    let ctx = tx_context::dummy();
-    
-}
+    public entry fun block_user(_:&OwnerCap , platform_info: &mut BmtPlatformDetails , user_addr : address ){
```

Comment:
ou are blocking the user, okay. But you haven't checked this validation in the 'buy_tickets' function I think.

Reply:
yeh i need to add that